data/ipl3.cfg

```diff
@@ -8,23 +8,47 @@ name = ipl3
 current = True
 phase = 5
 release_date = 2023-11-01
+SAS_BASE_DIR = %(FILESYSTEM)s
+SAS_ROOT = %(FILESYSTEM)s/%(name)s
 
-[BHM]
+[general]
 name = ipl-3
-IPL_ROOT = %(FILESYSTEM)s/%(name)s
-BOSS_SPECTRO_REDUX = %(IPL_ROOT)s/spectro/boss/redux
+SAS_BASE_DIR = %(FILESYSTEM)s
+SAS_ROOT = %(FILESYSTEM)s/%(name)s
 
-[MWM]
+[DATA]
 name = ipl-3
-IPL_ROOT = %(FILESYSTEM)s/%(name)s
-MWM_ASTRA= %(IPL_ROOT)s/spectro/astra
-MWM_SSPP= %(IPL_ROOT)s/vac/mwm/sspp
-APOGEE_REDUX = %(IPL_ROOT)s/spectro/apogee/redux
```

Comment:
why are these sections removed for IPL-3?  and why aren't these ini sections included in `dr19.cfg`

Reply:
Note in generaal, that the set of ini sections in both dr19.cfg and ipl3.cfg should be similar to dr18.cfg and not be based on sdsswork.cfg. i.e. the top level folders in dr19 (and ipl3) are listed at https://dr19.sdss.org/sas/dr19 and include [DATA], [MOS] (no change from dr18.cfg), [SPECTRO] and [VAC] only!